Documentation/cmdref/cilium_policy.md

```diff
@@ -25,6 +25,7 @@ Manage security policies
 ### SEE ALSO
 
 * [cilium](../cilium)	 - CLI
+* [cilium policy cache](../cilium_policy_cache)	 - Display cache mapping selectors to identities
```

Comment:
We also have a PolicyCache, should we name this API something different? E.g., reserve "cilium policy cache" for the PolicyCache introspection, and maybe "cilium policy selectors" for this one?

Reply:
Done.